appengine/go11x/tasks/create_task/create_task.go

```diff
@@ -61,6 +61,9 @@ func createTask(projectID, locationID, queueID, message string) (*taskspb.Task,
 		return nil, fmt.Errorf("cloudtasks.CreateTask: %v", err)
 	}
 
+	// Closing the client is important, if not done then it will result in Memory spike
+	defer client.Close()
```

Comment:
Will you move this line to right below the error check for `NewClient` above? It's possible there will be some other issue between creating the client and here, so we want to make sure the `defer` is actually set.

Reply:
Yes, sure